File: LeetCode/Simulation/JS/Leetcode1460-make-two-arrays-equal-by-reversing-sub-arrays.js
var canBeEqual = function(target, arr) {
    target.sort((a,b) => a - b);
    arr.sort((a,b) => a - b);
    const n = target.length;
    for(let i = 0;i < n;i++) {
        if(target[i] != arr[i]) {
            return false;
        }
    }
    return true;
};

File: LeetCode/Simulation/Golang/Leetcode1460-make-two-arrays-equal-by-reversing-sub-arrays.go
package sbgo

import "sort"

func canBeEqual(target []int, arr []int) bool {
	sort.Ints(target)
	sort.Ints(arr)
	for i := 0; i < len(arr); i++ {
		if arr[i] != target[i] {
			return false
		}
	}
	return true
}
